test/mocha/10-api.js

```diff
@@ -23,4 +24,14 @@ describe('bedrock-config-yaml', () => {
     config.loggers.console.someLoggerCombinedConfig
       .should.equal('FDRqpNJLVkgfVxPe');
   });
+  it('configuration can be loaded via environment variable', async () => {
+    process.env.BEDROCK_CONFIG = 'dGVzdC1iZWRyb2NrLWVudi15YW1sOgogIHRlc3RFb' +
+      'nZWYWx1ZTogMTIzMTIzMTIzMTIz';
+
+    applyConfigFromEnv();
+
+    config['test-bedrock-env-yaml'].should.eql({
```

Comment:
it would be good to assert that this value does *not* exist before running `applyConfigFromEnv`.